scanner/image/db/download.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -euo pipefail
+
+postgres_major=15
+pg_rhel_major=8
+
+arch="$(uname -m)"
+dnf_list_args=()
+if [[ "$arch" == "arm64" ]]; then
+  arch="aarch64"
+  # Workaround for local Darwin ARM64 builds due to "Error: Failed to download metadata for repo 'pgdg14': repomd.xml GPG signature verification error: Bad GPG signature"
+  dnf_list_args=('--nogpgcheck')
```

Comment:
Is this still relevant for this?

Reply:
Just realized I could just check `https://download.postgresql.org/pub/repos/yum/reporpms`. It's there. I removed this line, thanks